src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/BwaAndMarkDuplicatesPipelineSpark.java

```diff
@@ -54,13 +54,16 @@
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
     protected String output;
 
+    @Argument(fullName = MarkDuplicatesSpark.DO_NOT_MARK_UNMAPPED_MATES, doc = "Enabling this option will mean unmapped mates of duplicate marked reads will not be marked as duplicates.")
+    public boolean dontMarkUnmappedMates = false;
```

Comment:
Move this argument and `duplicates_scoring_strategy` into a `MarkDuplicatesSparkArgumentCollection`